romsel_aktheme/arm9/source/windows/rominfownd.cpp

```diff
@@ -480,9 +480,17 @@ void RomInfoWnd::setFileInfo(const std::string &fullName, const std::string &sho
 void RomInfoWnd::setRomInfo(const DSRomInfo &romInfo)
 {
     _romInfo = romInfo;
-
-    auto bannerTitle = _romInfo.banner().titles[ms().getGuiLanguage()];
-    _romInfoText = unicode_to_local_string(bannerTitle, 128, NULL);
+    
+	if (ms().getGuiLanguage() == 6)
```

Comment:
Don't special-case language 6. Maybe we can use a modulus here or a mapping from TWLMenu GUI Language to Nintendo language. Also this may not work for non iQue games